packages/mesa-vulkan-icd-freedreno-dri3/build.sh

```diff
@@ -0,0 +1,53 @@
+TERMUX_PKG_HOMEPAGE=https://www.mesa3d.org
+TERMUX_PKG_DESCRIPTION="An open-source implementation of the OpenGL specification"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_LICENSE_FILE="docs/license.rst"
+TERMUX_PKG_MAINTAINER="@termux"
```

Comment:
@xMeM Hello, could you please set maintainer to yourself?

Reply:
OK, thanks.